src/Form/Form.js

```diff
@@ -40,7 +40,7 @@ class Form extends Component {
     return (
       <form className="joke-form" alt="form">
```

Comment:
We should probably use some sort of linter to make all of our quotations consistent